.cirrus.yml

```
@@ -0,0 +1,14 @@
+freebsd_task:
+  freebsd_instance:
+    matrix:
+    - image: freebsd-12-4-release-amd64
+    - image: freebsd-13-2-release-amd64
+  build_script:
+    - mkdir build
+    - cd build
+    - ../configure
+    - make -j$(sysctl -n hw.ncpu)
+  test_script:
+    - cd build
+    # dtrace fails to build on FreeBSD - see gh-73263
+    - make test TESTOPTS="-u-network -x test_dtrace"
```

Look at ``.github/workflows/build.yml`` for other jobs running tests. You may also look at the way more complicated buildbot configuration: https://github.com/python/buildmaster-config/blob/main/master/custom/factories.py

```suggestion
    - make buildbottest TESTOPTS="-j2 -u-network -x test_dtrace"
```